src/pineko/evolve.py

```diff
@@ -46,6 +49,52 @@ def sv_scheme(tcard):
     return modsv
 
 
+def get_grid_convolution_type(kv):
+    """Retrieve the ekos convolution type.
+
+    Parameters
+    ----------
+    kv: dict
+        pineappl grid metadata
+    """
+    # TODO: This should probably changed in the future to use the Grid::convolutions
+    if "convolution_type_1" in kv:
+        conv_type_1 = kv["convolution_type_1"]
+    # TODO: polarized is now deprecated, needed for compatibility
+    elif "polarized" in kv and kv["polarized"]:
+        conv_type_1 = "PolPDF"
+    else:
+        conv_type_1 = "UnpolPDF"
+
+    # TODO: initial_state_2 is now deprecated, needed for compatibility
+    if "convolution_particle_2" in kv:
+        part_2 = kv["convolution_particle_2"]
```

Comment:
I wonder if instead of accounting here for the old keys, it would be possibile to do this in `pineappl`. I mean, making sure that if pineappl is asked for legacy keys, it knows which new key to give back. I do not like to have all these `if, elif, else` every time we update some convention somewhere.

Reply:
Of course I understand that this is just for the time being, but these choices tend to last forever when they work.

Reply:
I'm not sure if this is possible, as we added new functionalities, and here is where we are consuming them. 
But maybe @cschwan or @Radonirinaunimi have a better opinion.

Reply:
This could only be fully resolved once https://github.com/NNPDF/pineko/pull/181#discussion_r1689312568 is available (which will not be in the very close future). 

Similar the other items, we can keep this as it is for now and  open an issue to remind ourselves to update this once the feature is available.

Reply:
Yes, I agree. Let's open an issue for this